src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-relationships/GeneRelationships.tsx

```diff
@@ -48,34 +52,48 @@ const tabClassNames = {
 type Props = {
   isSidebarOpen: boolean;
   selectedTabName: GeneRelationshipsTabName | null;
-  setActiveGeneRelationshipsTab: (tab: string) => void;
+  push: Push;
 };
 
 const GeneRelationships = (props: Props) => {
+  const { genomeId, entityId } = useParams() as { [key: string]: string };
```

Comment:
Seems we are using this type (`{ [key: string]: string }`) in quite a few places. I wonder if we should have a shared type for this.

Reply:
Not sure how best to approach it. I am definitely being lazy here; it might be best to describe the params object for different applications, so that typescript know  that `genomeId`, and `entityId` are actually the expected properties of the params object for EntityViewer.